sharding/params/config.go

```diff
@@ -0,0 +1,40 @@
+// Package params defines important configuration options to be used when instantiating
+// objects within the sharding package. For example, it defines objects such as a
+// ShardConfig that will be useful when creating new shard instances.
+package params
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// DefaultShardConfig contains default configs for node to use in the sharded universe
+var DefaultShardConfig = ShardConfig{
```

Comment:
Please make this a pointer to a config, otherwise the value will be copied many times or the user will have to take the address of this variable to use it.